<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>340</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>

    <!--************************** VERTICAL SPACER **************************-->
    <item row="0" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>

    <item row="1" column="0">
     <widget class="QLabel" name="cmapLabel">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>CMap:</string>
      </property>
     </widget>
    </item>

    <item row="1" column="1">
     <widget class="QComboBox" name="cmap">
      <item>
       <property name="text">
        <string>gray</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Spectral</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>YlOrBr</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>hsv</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>hot</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Set1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>gist_earth</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>nipy_spectral</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>tab10</string>
       </property>
      </item>
     </widget>
    </item>

    <!--************************** VERTICAL SPACER **************************-->
    <item row="2" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>

    <item row="3" column="0">
     <widget class="QLabel" name="scaleRangeLabel">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Scale Range:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="scaleRange">
      <item>
       <property name="text">
        <string>Min-Max</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ZScale</string>
       </property>
      </item>
     </widget>
    </item>

    <!--************************** VERTICAL SPACER **************************-->
    <item row="4" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>

    <item row="5" column="0">
     <widget class="QLabel" name="scaleAlgorithmLabel">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Scale Algorithm:</string>
      </property>
     </widget>
    </item>

    <item row="5" column="1">
     <widget class="QComboBox" name="scaleAlgorithm">
      <item>
       <property name="text">
        <string>Linear</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Log</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Power</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Square Root</string>
       </property>
      </item>
      <item>
      <property name="text">
        <string>Sin</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ArcSin</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Histogram</string>
       </property>
      </item>
     </widget>
    </item>

    <!--************************** VERTICAL SPACER **************************-->
    <item row="6" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>

    <item row="7" column="0">
     <widget class="QLabel" name="removeNaNsLabel">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Remove NaNs:</string>
      </property>
     </widget>
    </item>

    <item row="7" column="1">
     <widget class="QCheckBox" name="removeNaNs">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>

    <!--************************** VERTICAL SPACER **************************-->
    <item row="8" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>

    <item row="9" column="0">
     <widget class="QLabel" name="showAllLabel">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Show All:</string>
      </property>
     </widget>
    </item>

    <item row="9" column="1">
     <widget class="QCheckBox" name="showAll">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>

    <!--************************** VERTICAL SPACER **************************-->
    <item row="10" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>

    <item row="11" column="0">
     <widget class="QLabel" name="show3DLabel">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Show 3D:</string>
      </property>
     </widget>
    </item>

    <item row="11" column="1">
     <widget class="QCheckBox" name="show3D">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>

    <!--************************** VERTICAL SPACER **************************-->
    <item row="12" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>

    <item row="13" column="0">
     <widget class="QLabel" name="minValueLabel">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Min Value:</string>
      </property>
     </widget>
    </item>

    <item row="13" column="1">
     <widget class="QLabel" name="minValue">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>

    <!--************************** VERTICAL SPACER **************************-->
    <item row="14" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>

    <item row="15" column="0">
     <widget class="QLabel" name="maxValueLabel">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Max Value:</string>
      </property>
     </widget>
    </item>

    <item row="15" column="1">
     <widget class="QLabel" name="maxValue">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>

    <!--************************** VERTICAL SPACER **************************-->
    <item row="16" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>

    <item row="17" column="0">
     <widget class="QLabel" name="meanCenterLabel">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Mean Center:</string>
      </property>
     </widget>
    </item>

    <item row="17" column="1">
     <widget class="QLabel" name="meanCenter">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>

    <!--************************** VERTICAL SPACER **************************-->
    <item row="18" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>

    <item row="19" column="0">
     <widget class="QLabel" name="showDifferenceLabel">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Show Difference:</string>
      </property>
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>

    <item row="19" column="1">
     <widget class="QCheckBox" name="showDifference">
      <property name="text">
       <string/>
      </property>
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>

    <!--************************** VERTICAL SPACER **************************-->
    <item row="20" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>

    <item row="21" column="0">
     <widget class="QLabel" name="invertDiffLabel">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Invert Difference:</string>
      </property>
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>

    <item row="21" column="1">
     <widget class="QCheckBox" name="invertDiff">
      <property name="text">
       <string/>
      </property>
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>

    <!---*******************************************************************-->

   </layout>
  </widget>


  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>340</y>
     <width>300</width>
     <height>60</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>

    <item>
     <widget class="QLabel" name="indexImageLabel">
      <property name="text">
       <string></string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="prevImageButton">
        <property name="text">
         <string>Previous Image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nextImageButton">
        <property name="text">
         <string>Next Image</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
